### PR TITLE
More robust handling of grace notes and beams

### PR DESCRIPTION
**Pull request by th-we**
More test cases and code fixing them.

### CODE REVIEW

File: mei2ly.xsl [hunk 1]
```diff
@@ -2478,14 +2484,25 @@
...
-    <xsl:text>\grace </xsl:text>
+  </xsl:template>
+  <xsl:template mode="closeGraceBeamGroup" match="mei:note/@grace[ancestor::mei:chord]" priority="10"/>
+  <xsl:template mode="closeGraceBeamGroup" match="@grace"/>
+  <xsl:template mode="closeGraceBeamGroup" match="@grace[(.., ../mei:note)[key('isGraceBeamEnd', generate-id())]]" priority="5">
+    <xml:text>}</xml:text>
   </xsl:template>
   <!-- set articulation -->
-  <xsl:template name="setArticulation">
+  <xsl:template name="setArticulation" mode="setArticulation" match="mei:artic|*[@artic]">
```
Comment (rettinghaus) [inline mei2ly.xsl:2505]: Did this got in by mistake? Seems to come from another branch.

**Review by rettinghaus — CHANGES_REQUESTED** [review]
revert `setArticulation`

**Comment by rettinghaus** [comment]
Will you work on this? Or sould I close the PR?

**Comment by th-we** [comment]
Good that you remind me. I believe I'll be forced to come back to this issue for processing our material. Other things are priority right now, though.

**Comment by rettinghaus** [comment]
Due to code conflicts I'm closing this for now.

**Pull request closed without merging**